source/user_guide/predictions.rst

```diff
@@ -32,23 +32,38 @@ The following recipe will supplement the content this guide will cover:
 
 Concepts & Terminology
 ----------------------
-**Regression** - is used to describe the relationship between one or more Context Features and a continuous numeric Action Feature, as in this guide predicting the **Highway MPG** of a vehicle based on it's physical characteristics and year manufactured.  
+**Regression** - is used to describe the relationship between one or more Context Features and a **continuous** numeric Action Feature, as in this guide predicting the **Highway MPG** of a vehicle based on it's physical characteristics and year manufactured.  
```

Comment:
it's = it is
its = possessive

I'm fairly confident about this, google if you wish. I think these should use "its" theres a couple spots here where it applies so keep an eye out for it